README.md

```diff
@@ -55,6 +57,41 @@ configured in `config/environments/development.rb`:
 curl http://localhost:3000/api/v1/2019/subjects.json -H "Authorization: Bearer bats"
 ```
 
+### V2
+
+#### Development
+
+In development mode, authenticating with V2 of the API relies on an email
+address of an existing use in the database being supplied as the bearer token.
+An example HTTP request would look like:
+
+```
+GET /api/v2/users.json
+Authorization: Bearer user@digital.education.gov.uk
+```
+
+or with curl:
+
+```bash
+curl http://localhost:3000/api/v2/users.json -H "Authorization: Bearer user@digital.education.gov.uk"
+```
+
+#### Production
+
+In production mode the bearer token is an HMAC-encrypted JWT with the JSON payload:
```

Comment:
I think encrypted is just fine, vs xyz encryped

Reply:
Are you worried about leaking security info?

Reply:
Let's not open ourselves to that as well as tying us to something in particular for no reason